src/Lumi/Components/Select.purs

```diff
@@ -415,27 +422,37 @@ styles = jss
               }
 
           , "&:hover lumi-select-input": lumiInputHoverStyles
-          , "&:focus lumi-select-input, & lumi-select-inner[data-focus=\"true\"] lumi-select-input": lumiInputFocusStyles
+
+          , "&:focus lumi-select-input, & lumi-select-inner[data-focus=\"true\"] lumi-select-input":
+              { extend: lumiInputFocusStyles
+              , "& lumi-select-input-placeholder":
+                  { display: "none"
+                  }
+              }
+
           , "&[data-disabled=\"true\"] lumi-select-input": lumiInputDisabledStyles
 
           , "& lumi-select-input-selected-values":
               { margin: "-2px" -- (gap)
 
               , display: "flex"
               , flexFlow: "row wrap"
-              , flex: "1"
+              , flex: "1 1 auto"
+              , minWidth: 0
 
               , "& > lumi-select-input-selected-value, & > lumi-select-input-placeholder, & > input.select-native-input":
                   { margin: "2px" -- (gap)
                   }
-              , "& > lumi-select-input-placeholder":
-                  { position: "absolute"
-                  }
               }
 
           , "& lumi-select-input-placeholder":
               { color: cssStringHSLA colors.black2
               , lineHeight: "32px"
+              , maxWidth: "calc(100% - 20px)"
+              , display: "block"
```

Comment:
This `calc` looks a bit dangerous to me or at least it could cause some maintenance problems in the future. Can't we just have `flex: 1` in it so it takes the rest of the space available (minus the width of `lumi-select-clear-icon`)?

Reply:
I'll check again in case I missed something, but that didn't seem to work.

Reply:
Oh hey, it works :sweat_smile: Maybe something was interfering when I tried earlier.